### DISCUSSION TITLE
Deadline extension

### DISCUSSION THREAD

**Discussion by maltanar** [Xilinx/brevitas-radioml-challenge-21#13] [Announcements]
We've decided to extend the submission deadline by 1 week. The submission deadline for the challenge is now **2021-10-09 AoE (anywhere on Earth)**.  The date for the announcement of results and winners remains unchanged (2021-10-27).

Please make sure to read the "Criteria for valid submissions" section under the "Evaluation criteria" tab on the [challenge website](https://bit.ly/brevitas-radioml-challenge-21) and don't hesitate to ask if you have any questions.

**Comment by bradstimpson** [comment]
@maltanar we are trying to submit our results on the ITU site and it doesn't look like we are permitted to any more.  Will the ITU site be updated to reflect the new deadline?

**Reply by maltanar** [reply]
Looks like there is an issue with the ITU portal, let me check this with the admins and post an update. You should be able to submit up until 9th October.

**Reply by maltanar** [reply]
Update: the admins have confirmed that there is an issue with the portal that is not trivial to fix, we're looking into alternative ways for collecting the submissions.

**Reply by Aakasha01Agarwal** [reply]
@maltanar did any solution come up?

**Reply by maltanar** [reply]
We're still discussing with the ITU folks to see how we can best resolve this in a way that can accommodate large zipfiles. Apologies for any inconvenience this is causing, we'll share an update as soon as we can.

**Reply by maltanar** [reply]
Google Forms-based submission link is now live, please see https://github.com/Xilinx/brevitas-radioml-challenge-21/discussions/18

**Reply by Aakasha01Agarwal** [reply]
Ah thank you so much. Just a suggestion, you guys should also put this link on the ITU website, there might be some people who don't follow this thread.
